pandas/_version.py

```diff
@@ -48,8 +49,8 @@ class NotThisMethod(Exception):
     pass
 
 
-LONG_VERSION_PY = {}
-HANDLERS = {}
+LONG_VERSION_PY = {}  # type: Dict
```

Comment:
Is this even being used anywhere? If it's dead code I think OK to remove

Reply:
OK we should really minimize the use of `Any` though. It's possible to be more strict about annotation in this instance

Reply:
If you can point me to the file that is using `LONG_VERSION_PY` maybe then I can figure out the type of it's content.

Reply:
You should double check but I think it is dead code so OK to remove. Try locally and run tests to confirm

Reply:
`LONG_VERSION_PY` looks dead and I have commented it. Sorry can't test locally because no matter whatever I do VS Code Build Tools won't install on my laptop.

Reply:
All tests all passing after commenting `LONG_VERSION_PY` so I will remove it.